code/modules/robotics/bot/secbot.dm

```diff
@@ -354,7 +602,7 @@ Report Arrests: <A href='?src=\ref[src];operation=report'>[report_arrests ? "On"
 				src.mode = SECBOT_HUNT
 			..()
 
-	proc/navigate_to(atom/the_target,var/move_delay=3,var/adjacent=0)
+	proc/navigate_to(atom/the_target,var/move_delay=1,var/adjacent=0)
```

Comment:
*concern*

Reply:
I forgot I set that. Couldn't tell if it did anything. I'll fix it.